master/internal/core.go

```diff
@@ -1084,12 +1084,12 @@ func (m *Master) Run(ctx context.Context, gRPCLogInitDone chan struct{}) error {
 		return err
 	}
 
-	command.RegisterAPIHandler(
-		m.system,
-		m.echo,
-		m.db,
-		m.rm,
-	)
+	// Wait for all NTSC services to initialize.
+	command.SetDefaultCmdService(m.db, m.rm)
+	// Restore any commands.
+	if err = command.DefaultCmdService.RestoreAllCommands(ctx); err != nil {
```

Comment:
it is a style thing, but i'd rather have a NewCommandService and constructs the command service and makes sure it is restored before we call SetDefaultCmdService to it is compiler verified no one can call the default command service before it is correctly instantiated